hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/SystemFileUpdateFacility.java

```diff
@@ -109,6 +111,9 @@ public void handleTxBody(@NonNull final HederaState state, @NonNull final Transa
             } else if (fileNum == config.upgradeFileNumber()) {
                 logger.error("Update of file number not implemented");
             }
+        } catch (final HandleException e) {
```

Comment:
As I mentioned already, we can simplify this. Can be done later, too.

Reply:
ok thank you